src/core/api/http-common.ts

```diff
@@ -2,15 +2,15 @@ import axios, { AxiosResponse } from 'axios';
 import { authToken, notValidRefreshToken, notValidToken } from '@/core/interceptors';
 
 export const HTTP = axios.create({
-  baseURL: 'https://home-project-academy.herokuapp.com/api/0',
```

Comment:
need to change back to https protocol, like it was before (http -> https lines 5 & 13)